packages/cozy-client/src/hooks/useQuery.spec.jsx

```diff
@@ -43,6 +44,55 @@ describe('use query', () => {
       })
     })
   })
+
+  it('fetches more results with bookmark', async () => {
+    const bookmark = 'bookmark-123'
+    const queryDefinition = Q('io.cozy.simpsons').offsetBookmark(bookmark)
+    const {
+      hookResult: {
+        result: { current }
+      },
+      client
+    } = setupQuery({
+      queryOptions: {
+        as: 'simpsonsBookmarked'
+      },
+      queryDefinition: () => queryDefinition,
+      storeQueries: {
+        simpsonsBookmarked: {
+          data: simpsonsFixture,
+          doctype: 'io.cozy.simpsons',
+          definition: queryDefinition
```

Comment:
There is no bookmark here ? It seemed to me that we'd use off with bookmark only if the query result contained a bookmark ?

Reply:
Yep, there is a book mark in the definition on l. 50!

Reply:
After investigating, we found it's actually using `skip` in this test too, but *also* the bookmark from the definition. I'm going to patch up the test.